{{cookiecutter.project_name}}/pyproject.toml

```diff
@@ -27,14 +27,11 @@ flake8-bandit = "^4.1.1"
 flake8-broken-line = "^1.0.0"
 flake8-comprehensions = "^3.14.0"
 flake8-debugger = "^4.1.2"
-flake8-docstrings = "^1.7.0"
 flake8-eradicate = "^1.5.0"
 flake8-isort = "^6.1.1"
 flake8-quotes = "^3.4.0"
-flake8-string-format = "^0.3.0"
 flake8-pyproject = "^1.2.3"
```

Comment:
Maybe not necessary, because separate `.flake8` config file is a better if config is big (100+ lines)